Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -1,13 +1,17 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
  *
```

Comment:
Detailed review needed.

Reply:
Had to revert to enable auto-grow. Work remains to ensure TextInput can grow correctly.

---
In reply to: [169647991](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169647991) [](ancestors = 169647991)